java/java.lsp.server/nbcode/nbproject/platform.properties

```diff
@@ -344,6 +343,7 @@ disabled.modules=\
     org.netbeans.modules.j2ee.sun.appsrv,\
     org.netbeans.modules.j2ee.sun.dd,\
     org.netbeans.modules.j2ee.sun.ddui,\
+    org.netbeans.modules.jakartaee.web.beans,\
```

Comment:
This looks strange (both changes). Unintentional?

Reply:
I will certainly review this adjustment.